modules/auth_plugins/app/views/hooks/login/_providers_css.html.erb

```diff
@@ -36,7 +36,7 @@ See COPYRIGHT and LICENSE files for more details.
       .op-app-header #nav-login-content .login-auth-providers a.auth-provider.auth-provider-<%= pro[:name] %> {
         background-image:  url('<%= asset_path(pro[:icon]) %>') ;
       }
-      .op-modal .login-auth-providers a.auth-provider.auth-provider-<%= pro[:name] %> {
+      .spot-modal .login-auth-providers a.auth-provider.auth-provider-<%= pro[:name] %> {
```

Comment:
I'd like to avoid the `spot-modal` class to be overwritten somewhere outside. Please add a custom class to that modal, which you can reuse here.

Reply:
selector without `spot-modal` is strong enough at this place. Removed `.spot-modal`